ivy/functional/ivy/creation.py

```diff
@@ -382,22 +408,31 @@ def eye(
     n_rows
         number of rows in the output array.
     n_cols
-        number of columns in the output array. If None, the default number of columns in
-        the output array is equal to n_rows. Default: None.
+        number of columns in the output array.
+        If None, the default number of columns in the output array is
+        equal to n_rows.Default: None.
     k
-        index of the diagonal. A positive value refers to an upper diagonal, a negative
-        value to a lower diagonal, and 0 to the main diagonal. Default: 0.
+        index of the diagonal.
+        A positive value refers to an upper diagonal,
+        a negative value to a lower diagonal,
+        and 0 to the main diagonal. Default: 0.
     dtype
-        output array data type. If dtype is None, the output array data type must be the
-        default floating-point data type. Default: None.
+        output array data type.
+        If dtype is None, the output array data type must be the default floating-
+        point data type. Default: None.
     device
          device on which to place the created array.
 
     Returns
     -------
     ret
-        device on which to place the created array. Default: None.
+        a two-dimensional array with ones on the k h diagonal and zeros elsewhere.
 
+    Examples
+    --------
+    >>> y = ivy.eye(2)
+    >>> print(y)
+    ivy.array([[1, 0],[0, 1]])
```

Comment:
I think the examples are still not exhaustive. Please refer [here](https://lets-unify.ai/ivy/contributing/13_docstring_examples.html)